toxcore/DHT.h

```diff
@@ -195,20 +191,15 @@ int packed_node_size(Family ip_family);
  * @return size of packet on success.
  * @retval -1 on failure.
  */
-non_null()
-int dht_create_packet(const Memory *mem, const Random *rng,
-                      const uint8_t public_key[CRYPTO_PUBLIC_KEY_SIZE],
-                      const uint8_t *shared_key, uint8_t type,
-                      const uint8_t *plain, size_t plain_length,
-                      uint8_t *packet, size_t length);
+int dht_create_packet(non_null() const Memory *mem, non_null() const Random *rng, non_null() const uint8_t public_key[CRYPTO_PUBLIC_KEY_SIZE], non_null() const uint8_t *shared_key, uint8_t type,
+                      non_null() const uint8_t *plain, size_t plain_length, non_null() uint8_t *packet, size_t length);
```

Comment:
I think the formatting before was nicer.
It's kind of a downside of adding the attribute to every parameter.

Reply:
Ok. What action do you want taken here?

Reply:
Is a line per argument too much?